changelog.md

```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## 0.27.1
+
+### Added
+
+- System tray executable now populates list of local projects
+  ([PR #1293](https://github.com/ActiveState/cli/pull/1293))
+
```

Comment:
Let's leave this off since we're not actually distributing anything. ie. version 0.27.1 (actually we're at 0.28.0 now..) won't ship with the systray yet, even though it's part of the codebase.